lib/plugins/track.js

```diff
@@ -189,7 +198,7 @@ exports.configure = function configure (config) {
 
         // Break when cookies aren't being cleared and it gets set properly
         // Don't break when uuid is falsy so all the cookies get cleared
-        if (cookie.getItem(storage.name) && uuid) {
+        if (uuid && cookie.getItem(storage.name) === uuid) {
```

Comment:
On Second thought, i think that if we reject/removeCookie if uuid is null, we can simplify this if with just the latter part